.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,40 @@
+name: "Integration test"
+# NOTE: We're just running the tests that require earthdata login here; we
+#       don't distinguish yet between unit and integration tests yet.
```

Comment:
```suggestion
# NOTE: We're just running the tests that require earthdata login here; we
#       don't distinguish between unit and integration tests yet.
```

Is your thought that we'll eventually transition to unit + integration, so it makes sense to rename to those now?

Reply:
Yes, that's my thinking. It's common terminology that is recognizable at a glance to those familiar with it, and also searchable. I'm not attached to this naming, however!

I would like to, if there's time, eventually split our tests up in to "unit" and "integration" directories.

Reply:
It auto-resolved this when I accepted your suggestion, so I un-resolved. You can mark resolved if you feel that's the case :)

Reply:
I'm fine with the rename, I just wasn't sure if they were "accurate" titles given we haven't separated the tests yet.

Reply:
There are definitely some things I would call "integration tests" that will run in the unit tests job. I didn't want to make this PR huge by actually digging in to that though :)

Reply:
> There are definitely some things I would call "integration tests" that will run in the unit tests job. I didn't want to make this PR huge by actually digging in to that though :)

That's why I was wondering aloud if it's a misnomer to rename before we've done the separation. I don't have strong feelings about it but didn't want to create a more-confusing situation for someone else in the future.

Reply:
How do you feel about the priority of separating the tests? If we're not going to do it anytime soon, perhaps different names are appropriate. Perhaps just "Tests" and "Behind EDL tests"? :shrug:

Reply:
> How do you feel about the priority of separating the tests?

Medium. Honestly, I think I'd prefer to have more tests/better test coverage. I'm guessing that ultimately we might end up with three categories: behind EDL, unit, and integration. So renaming as you suggest "tests" and "behind EDL tests" lets us move forward with later separating out things in "tests" as needed.

Reply:
I've been thinking of "behind EDL" as a subset of "integration", and using for example [skipif](https://docs.pytest.org/en/stable/how-to/skipping.html#skipif) to control whether those tests run in certain environments. What do you think?

Reply:
Those sound like reasonable approaches (and we can adjust the triggers for the integration tests more broadly then).